Branch-SDK/src/io/branch/indexing/BranchUniversalObject.java

```diff
@@ -80,6 +105,8 @@ public BranchUniversalObject() {
         type_ = "";
         indexMode_ = CONTENT_INDEX_MODE.PUBLIC; // Default content indexing mode is public
         expirationInMilliSec_ = 0L;
+        price_ = 0.0;
+        currency_ = CURRENCY_TYPE.NONE;
```

Comment:
The problem I have with that is as I mention below: it causes the Android SDK to send empty strings for values, where the iOS SDK doesn't send anything, which pollutes the backend and has us store unnecessary data.